### PR TITLE
Added support to the operator detect_text_in_image_tesseract for Tamil and Telugu

### PR DESCRIPTION
**Pull request by plon-Susk7**
Solved issue #322 

1. Made changes to Dockerfile to install tesseract-ocr for tamil and telugu.
2. Added test cases for tamil and telugu in test_detect_text_in_image_tesseract.py

### CODE REVIEW

File: src/core/operators/detect_text_in_image_tesseract.py [hunk 1]
```diff
@@ -29,11 +30,11 @@ def state():
     pass
 
 
-# if __name__ == "__main__":
-#     initialize(param={})
-#     image_path = 'sample_data/hindi-text.png'
-#     text_data = run(image_path)
-#     print(text_data)
+if __name__ == "__main__":
```
Comment (aatmanvaidya) [inline src/core/operators/detect_text_in_image_tesseract.py:33]: lets remove the if __name__ from the code

Comment (plon-Susk7) [reply src/core/operators/detect_text_in_image_tesseract.py]: Done!

File: src/Dockerfile [hunk 2]
```diff
@@ -31,9 +31,9 @@
 RUN pip install --no-cache-dir --require-hashes --no-deps -r /usr/app/base_requirements.txt
 
 RUN apt-get update --fix-missing && apt-get -y upgrade && apt-get install -y --no-install-recommends vim curl ffmpeg
-# RUN apt-get update && \
-#     apt-get -y upgrade && \
-#     apt-get install -y tesseract-ocr tesseract-ocr-hin
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y tesseract-ocr tesseract-ocr-hin tesseract-ocr-tam tesseract-ocr-tel
```
[the remark this answers is not in the document]
Comment (aatmanvaidya) [reply src/Dockerfile:36]: @plon-Susk7 can you fix this warning too by just adding a `--no-install-recommends` to the line

[the remark this answers is not in the document]
Comment (plon-Susk7) [reply src/Dockerfile]: sorry missed this error. I have made the changes.

**Pull request merged**